types/src/shared/utils/iots_utils.ts

```diff
@@ -16,3 +16,16 @@ export function ioTsEnum<EnumType>(
     t.identity
   );
 }
+
+export interface BrandedRange {
+  readonly Range: unique symbol;
+}
+
+// Defines a function to generate a branded codec for validating numbers within a specific range.
```

Comment:
not familiar with those, but seems a tad verbose to validate a number between 1 and 100. Is this the standard way?

Reply:
It's io-ts standard way 🤓.